scss/_base_code.scss

```diff
@@ -1,5 +1,7 @@
 @import 'settings';
 
+$color-pre-bg: rgba($color-x-dark, 0.03);
```

Comment:
isthis a new variable? or is this what is used in the pr merged yesterday?

Reply:
Yeah, this is what was used there - I moved it here and changed the name for this PR, previously it was more obviously specific to the code snippet.